templates/containers/lxd.html

```diff
@@ -57,19 +57,43 @@ <h2>Why use LXD?</h2>
     <h2 class="p-muted-heading u-align--center">Supported guest operating systems</h2>
     <ul class="p-inline-images p-inline-images--large no-margin">
       <li class="p-inline-images__item">
-        <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}97e0563e-redhat.svg" alt="Redhat logo" />
+        <img class="p-inline-images__logo" src="https://alpinelinux.org/alpinelinux-logo.svg" alt="Alpine-Linux logo" />
```

Comment:
I think it would be better to upload this logo to the asset server and serve it from there - we can't rely on it always being available on a domain we don't control.